src/Amazon.Common.DotNetCli.Tools/Assets/AmazonCommonDotNetCliTools.targets

```diff
@@ -0,0 +1,10 @@
+<Project>
```

Comment:
Why do need to do have our own targets file instead of just using the CLI `dotnet msbuild -getProperty:OutputType:TargetFramework,TargetFrameworks` command?

Reply:
```
dotnet msbuild .\testapps\TestFunction\TestFunction.csproj -getProperty:OutputType:TargetFramework,TargetFrameworks
```

was returning 

```
{
  "Properties": {
    "OutputType:TargetFramework": "",
    "TargetFrameworks": ""
  }
}
``` 
for me. even though the csproj file looks like

```
<Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <TargetFramework>net6.0</TargetFramework>
    <AssemblyName>TestFunction</AssemblyName>
    <OutputType>Library</OutputType>
    <PackageId>TestFunction</PackageId>
    <GenerateAssemblyConfigurationAttribute>false</GenerateAssemblyConfigurationAttribute>
    <GenerateAssemblyCompanyAttribute>false</GenerateAssemblyCompanyAttribute>
    <GenerateAssemblyProductAttribute>false</GenerateAssemblyProductAttribute>
  </PropertyGroup>
  <ItemGroup>
    <PackageReference Include="Amazon.Lambda.Serialization.Json" Version="2.0.0" />
  </ItemGroup>
</Project>
```

Reply:
looks like it has to be `dotnet msbuild .\testapps\TestFunction\TestFunction.csproj  --getProperty:TargetFramework`. ill play around with this and see if the built in one works

Reply:
You should be using `TargetFramework` not `OutputType:TargetFramework`. The `OutputType` was used before when we were trying to see if the project was a class library or an executable.

Reply:
should be fixed now